pkgs/development/web/nodejs/v20.nix

```
@@ -0,0 +1,22 @@
+{
+  callPackage,
+  openssl,
+  fetchpatch,
+  python3,
+  enableNpm ? true,
+}: let
```

```suggestion
{ callPackage, openssl, fetchpatch, python3, enableNpm ? true }:

let
```

I think the formatting should match what's been done for the other versions.

---

I'd argue it really does not matter, as we'll eventually have consistent formatting treewide. 🤷‍♀️

---

It was applied automatically by my IDE using alejandra. I've reverted it to be inline with the other node packages. I've got my fingers crossed for RFC0101 🤞